### PR TITLE
Merge feature/131-locator

### PR DESCRIPTION
**Pull request by ercanozkaya**
see: #131

### CODE REVIEW

File: code/libraries/joomlatools/component/koowa/class/locator/plugin.php [hunk 1]
```diff
@@ -82,7 +82,15 @@ public function locate($classname, $basepath = null)
                 $basepath = $this->getNamespace($namespace);
             }
 
-            return $basepath.'/'.$package.'/'.$path.'.php';
+            $path = implode('/', $parts);
```
Comment (ercanozkaya) [inline code/libraries/joomlatools/component/koowa/class/locator/plugin.php]: @amazeika `$parts` is empty for a class like `PlgKoowaAbstract`

This leads to the following double slash issue: `/var/www/test/libraries/joomlatools/plugin/koowa//abstract.php`

Can we solve that too?

Comment (amazeika) [reply code/libraries/joomlatools/component/koowa/class/locator/plugin.php]: @ercanozkaya We can clean it up to make it pretty, but double slashes aren't a problem are they?

Comment (amazeika) [reply code/libraries/joomlatools/component/koowa/class/locator/plugin.php]: @ercanozkaya Done handling the double slashes. Can you double check please?

**Comment by amazeika** [comment]
@ercanozkaya Can we merge this?

**Pull request merged**